dev/com.ibm.ws.concurrent_fat_jakarta/test-applications/ConcurrencyTestWeb/src/test/jakarta/concurrency/web/ConcurrencyTestServlet.java

```diff
@@ -1092,6 +1096,149 @@ public void testEJBDDManagedThreadFactoryDefinition() throws Exception {
+            if (!IOException.class.equals(exception.getClass()))
+                throw exception;
+
+            // Future.exceptionNow on running task:
+            try {
+                exception = (Throwable) exceptionNow.invoke(task3future);
+                if (exception == null)
+                    throw new AssertionError("exceptionNow returned null for running task");
+                else
+                    throw new AssertionError("exceptionNow returned value for running task").initCause(exception);
+            } catch (InvocationTargetException x) {
+                if (!(x.getCause() instanceof IllegalStateException))
+                    throw x;
+            } catch (IllegalStateException x) {
+                // pass
+            }
+
+            // Future.exceptionNow on task that is forcibly completed:
+            CompletableFuture<Long> task4future = executor.supplyAsync(() -> 664L);
+            task4future.complete(644L);
+            try {
+                exception = (Throwable) exceptionNow.invoke(task4future);
+                if (exception == null)
+                    throw new AssertionError("exceptionNow returned null for successfully completed task");
+                else
+                    throw new AssertionError("exceptionNow returned value for successfully completed task").initCause(exception);
+            } catch (InvocationTargetException x) {
+                if (!(x.getCause() instanceof IllegalStateException))
+                    throw x;
+            } catch (IllegalStateException x) {
+                // pass
+            }
+
+            // Future.exceptionNow after obtruding the result with exceptional completion
+            task4future.obtrudeException(new SQLException("Not a real error."));
+            exception = (Throwable) exceptionNow.invoke(task4future);
+            if (!SQLException.class.equals(exception.getClass()))
+                throw exception;
+
+            // Future.exceptionNow after obtruding the exceptional result back to successful, but with a different value:
+            task4future.obtrudeValue(640L);
+            try {
+                exception = (Throwable) exceptionNow.invoke(task4future);
+                if (exception == null)
+                    throw new AssertionError("exceptionNow returned null for successfully completed task");
+                else
+                    throw new AssertionError("exceptionNow returned value for successfully completed task").initCause(exception);
+            } catch (InvocationTargetException x) {
+                if (!(x.getCause() instanceof IllegalStateException))
+                    throw x;
+            } catch (IllegalStateException x) {
+                // pass
+            }
+
+            // Future.resultNow on cancelled task:
+            assertEquals(true, task3future.cancel(true));
+            try {
+                exception = (Throwable) exceptionNow.invoke(task3future);
```

Comment:
Comment say's `resultNow` but we are still testing `exceptionNow`. 
I assume this is a typo in the comment, but if you do intend to test resultNow the test will need to be updated.

Reply:
Yes, the comment is a mistake (actually a copy/paste error). Thanks for spotting it.